tests/pipelines/test_pipelines_image_classification.py

```diff
@@ -177,6 +177,34 @@ def test_custom_tokenizer(self):
 
         self.assertIs(image_classifier.tokenizer, tokenizer)
 
+    @require_torch
+    def test_torch_float16_pipeline(self):
+        import torch
+
```

Comment:
Instead of importing here - we should import at the top of the module with: 

```py
if is_torch_available():
    import torch
```

```suggestion
```